settings.php

```diff
@@ -41,4 +41,8 @@
     $settings->add(new admin_setting_configcheckbox('logstore_xapi/backgroundmode',
         get_string('backgroundmode', 'logstore_xapi'),
         get_string('backgroundmode_desc', 'logstore_xapi'), 0));
+
+    $settings->add(new admin_setting_configtext('logstore_xapi/maxbatchsize',
```

Comment:
Point of interest: Watershed requests with a limit of 500 statements when pulling statements from another LRS. I'd actually be happy with a default batch size of unlimited and then people can introduce a batch size if required.